addPost.php

```diff
@@ -1,135 +1,199 @@
-                //Also for the future: improving the User interface as they recieve the message and redirect to other page is worth looking into
-            exit();
+          header("Location: addPost.php");
         }
     }
-
 ?>
-<!DOCTYPE html>
-    <html lang="en">
-        <head>
-            <meta charset="UTF-8">
-            <meta http-equiv="X-UA-Compatible" content="IE=edge">
-            <meta name="viewport" content="width=device-width, initial-scale=1.0">
-            <title>Document</title>
-            <link rel="stylesheet" href="style.css">
-
-        </head>
-        <body>
-            <div>
-                <form action="" method = 'post'> 
-                    <div class = 'flexContainer'>
-                        <div class = 'row1Container'>
-                            <div class = 'row1'>
-                                <div class="label">Posting Title</div>
-                                <input type="text" class = 'field' name = 'postingTitle' placeholder = 'ex: MiniFridge' required><br/>
-                            </div>
-                            <div class = 'row1'>
-                                <div class="label">Price</div>
-                                $<input type="number" class = 'field' name = 'price' placeholder = '' required><br/>
-                            </div>
 
-                            <div class = 'row1'>
-                                <div class="label">Location</div>
-                                <input type="text" class = 'field' name = 'location' placeholder = 'ex: Umrath Dorm' required><br/>
-                            </div>
-
-                            <div class = 'row1'>
-                                <div class="label">ZipCode</div>
-                                <input type="text" class = 'field' name = 'zipCode' placeholder = '' required><br/>
+<body>
+  <main>
+    <div class="container">
+      <header class="d-flex flex-wrap align-items-center justify-content-center justify-content-md-between py-3 mb-4 border-bottom">
+        <div class="col-md-3 mb-2 mb-md-0">
+          <a href="" class="d-inline-flex link-body-emphasis text-decoration-none">
+            <img src="/images/random.png" width ="230" height = "56" class = 'specific' alt="Picture">
+          </a>
+        </div>
+        <ul class="nav col-12 col-md-auto mb-2 justify-content-center mb-md-0">
+          <li><a href="#" class="nav-link px-2 link-secondary">Home</a></li>
+          <li><a href="#" class="nav-link px-2">Shop</a></li>
+          <li><a href="#" class="nav-link px-2">Features</a></li>
+          <li><a href="#" class="nav-link px-2">FAQs</a></li>
+          <li><a href="#" class="nav-link px-2">About</a></li>
+        </ul>
+        <div class="col-md-3 text-end">
+        <?php echo '<a href="logout.php"><button type="button" class="btn btn-primary">Logout</button></a>';?>
+        </div>
+      </header>
+    </div>
+  </main>
+<!--SECOND HEADER-->
```

Comment:
I think these can go in a function like `PrintHeader()` or `NavBar()` or whatever